src/WPF/ArcGISRuntime.WPF.Viewer/Samples/Layers/RasterColormapRenderer/RasterColormapRenderer.xaml.cs

```diff
@@ -0,0 +1,81 @@
+﻿// Copyright 2019 Esri.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at: http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific
+// language governing permissions and limitations under the License.
+
+using ArcGISRuntime.Samples.Managers;
+using Esri.ArcGISRuntime.Mapping;
+using Esri.ArcGISRuntime.Rasters;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Drawing;
+using System.Linq;
+using System.Windows;
+
+namespace ArcGISRuntime.WPF.Samples.RasterColormapRenderer
+{
+    [ArcGISRuntime.Samples.Shared.Attributes.Sample(
+        "Raster colormap renderer",
+        "Layers",
+        "Demonstrates how to use a colormap renderer on raster layer.",
```

Comment:
You'll want to update this to match the common readme text, **Apply a colormap renderer to a raster.**